content/docs/guides/aws-lambda.md

```diff
@@ -0,0 +1,273 @@
+
+    Deploying aws-node-project to stage dev (us-east-1, "default" provider)
+
+    ✔ Service deployed to stack aws-node-project-dev (71s)
+
+    dashboard: https://app.serverless.com/myorg/apps/my-aws-node-project/aws-node-project/dev/us-east-1
+
+    functions:
+      hello: aws-node-project-dev-hello (225 kB)
+
+    What next?
+    Run these commands in the project directory:
+
+    serverless deploy    Deploy changes
+    serverless info      View deployed endpoints and resources
+    serverless invoke    Invoke deployed functions
+    serverless --help    Discover more commands
+    ```
+
+4. Navigate to the `aws-node-project` directory created by the previous step and install the `node-postgres` package, which you will use to connect to the database.
+
+    ```bash
+    npm install pg
+    ```
+
+    After installing the `node-postgres` package, the following dependency should be defined in your `package.json` file:
+
+    ```json
+    {
+     "dependencies": {
+       "pg": "^8.8.0"
+     }
+    }
+    ```
+
+5. In the `aws-node-project` directory, add a `users.js` file, and add the following code to it:
+
+    ```js
+    'use strict';
+
+    const { Client } = require('pg');
+
+    module.exports.getAllUsers = async () => {
+     var client = new Client(process.env.DATABASE_URL);
+     client.connect();
+     var { rows } = await client.query('SELECT * from users');
+     return {
+       statusCode: 200,
+       body: JSON.stringify({
+         data: rows,
+       }),
+     };
+    };
+    ```
+
+    The code in the `users.js` file exports the `getAllUsers` function, which retrieves all rows from the `users` table and returns them as a `JSON` object in the `HTTP` response body.
+
+    This function uses the `pg` library to connect to the Neon database. It creates a new `Client` instance and passes the database connection string, which is defined in the `DATABASE_URL` environment variable. It then calls `connect()` to establish a connection to the database. Finally, it uses the `query()` method to execute a `SELECT` statement that retrieves all rows from the `users` table.
+
+    The query method returns a `Promise` that resolves to an object containing the rows retrieved by the `SELECT` statement, which the function parses to retrieve the `rows` property. Finally, the function returns an `HTTP` response with a status code of 200 and a body that contains a `JSON` object with a single `data` property, which is set to the value of the rows variable.
+
+6. Add the `DATABASE_URL` environment variable and the function definition to the `serverless.yml` file, which is located in your `aws-node-project` directory.
+
+    You can copy the connection string from **Connection Details** widget the Neon Console. Add the `DATABASE_URL` under `environment`, and add `sslmode=require` to the end of the connection string to enable SSL. The `sslmode=require` option tells PostgreSQL to use SSL encryption and verify the server's certificate.
```

Comment:
Not sure if we should include how to load environment variables from a `.env` file 🤔

https://www.serverless.com/framework/docs/environment-variables. I found this guide in Serverless framework's docs but I haven't tried it

Reply:
Add a note about using a .env file for environment variables.